webapp-django/crashstats/supersearch/jinja2/supersearch/search.html

```diff
@@ -24,6 +24,8 @@
     </div>
 
     <section id="search-form">
+        <p class="message warning">As of <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1114591">bug 1114591</a>, we have switched the meanings of the operators <code>$</code> (now <code>ends with</code>) and <code>^</code> (now <code>starts with</code>) so that they match the POSIX norm. Any search you saved using those operators will not work anymore, please update them. See the bug for more details. </p>
```

Comment:
We need a date here. We'll roughly pick one. Something that reflects when this feature change took place. E.g. Feb 9 2016.